commands/register.go

```diff
@@ -0,0 +1,49 @@
+package commands
+
+import (
+	"strings"
+
+	"github.com/Strum355/log"
+	"github.com/bwmarrin/discordgo"
+	"github.com/spf13/viper"
+)
+
+var helpStrings = make(map[string]string)
+var commandsMap = make(map[string]func(*discordgo.Session, *discordgo.MessageCreate))
+
+func command(name string, helpMessage string, function func(*discordgo.Session, *discordgo.MessageCreate)) {
+	helpStrings[name] = helpMessage
+	commandsMap[name] = function
+}
+
+//Register commands
+func Register(s *discordgo.Session) {
+	command("ping", "pong!", ping)
+	command("help", "displays this message", help)
+	command("register", "registers you as a member of the server", serverRegister)
+
+	s.AddHandler(messageCreate)
```

Comment:
Would we be able to add a handler to call `serverRegister` when a new users joins `discord.servers.PublicServer`